airbyte-integrations/connectors/source-hubspot/source_hubspot/manifest.yaml

```diff
@@ -0,0 +1,279 @@
+    type: SimpleRetriever
+    requester:
+      $ref: "#/definitions/base_requester"
+    record_selector:
+      $ref: "#/definitions/base_selector"
+    decoder:
+      type: JsonDecoder
+
+  stream_base:
+    type: DeclarativeStream
+    primary_key:
+      - "{{ parameters.get('primary_key', 'id') }}"
+    schema_loader:
+      $ref: "#/definitions/schema_loader"
+    retriever:
+      $ref: "#/definitions/base_retriever"
+
+  stream_without_pagination:
+    $ref: "#/definitions/stream_base"
+    retriever:
+      $ref: "#/definitions/retriever_without_pagination"
+
+  # Stream Definitions
+  marketing_emails_stream:
+    $ref: "#/definitions/stream_base"
+    $parameters:
+      name: marketing_emails
+      path: /marketing-emails/v1/emails/with-statistics
+      extractor_field_path: objects
+    transformations:
+      - type: AddFields
+        fields:
+          - path: ["rootMicId"]
+            value: "{{ record['rootMicId'] | string }}"
+
+  email_subscriptions_stream:
+    $ref: "#/definitions/stream_without_pagination"
+    $parameters:
+      name: email_subscriptions
+      path: /email/public/v1/subscriptions
+      extractor_field_path: subscriptionDefinitions
+
+  contacts_form_submissions_stream:
+    $parameters:
+      name: contacts_form_submissions
+      path: /contacts/v1/lists/all/contacts/all
+      primary_key: canonical-vid
+    $ref: "#/definitions/stream_base"
+    retriever:
+      type: SimpleRetriever
+      requester:
+        $ref: "#/definitions/base_requester"
+        request_parameters:
+          formSubmissionMode: all
+      record_selector:
+        type: RecordSelector
+        transform_before_filtering: true
+        record_filter:
+          type: RecordFilter
+          condition: "{{ record['timestamp'] >= format_datetime(config.get('start_date', '2006-06-01T00:00:00Z'), '%s', '%Y-%m-%dT%H:%M:%SZ')}}"
+        extractor:
+          type: CustomRecordExtractor
+          class_name: source_hubspot.components.ListFieldToSingleRecordByPkExtractor
+          field_path:
+            - contacts
+          pk_field_path:
+            - canonical-vid
+          related_field_path:
+            - form-submissions
+      paginator:
+        $ref: "#/definitions/vid_offset_cursor_paginator"
+      decoder:
+        type: JsonDecoder
+    transformations:
+      - type: DpathFlattenFields
+        delete_origin_value: true
+        replace_record: true
+        field_path:
+          - form-submissions
+
+  contacts_merged_audit_stream:
+    primary_key: canonical-vid
+    $parameters:
+      name: contacts_merged_audit
+      path: /contacts/v1/lists/all/contacts/all
+      primary_key: canonical-vid
+    $ref: "#/definitions/stream_base"
+    retriever:
+      type: SimpleRetriever
+      requester:
+        $ref: "#/definitions/base_requester"
+      record_selector:
+        type: RecordSelector
+        transform_before_filtering: true
+        record_filter:
+          type: RecordFilter
```

Comment:
maybe this is a question for later post-migration, but it seems like this should've just been a client side incremental stream

Reply:
yes, it it should be client-side incremental in this case. I migrated current stream's implementation. Should I create an issue to track streams like this and update them after low-code migration?